sdk/search/azure-search-documents/azure/search/documents/_internal/_search_client.py

```diff
@@ -100,6 +101,15 @@ def close(self):
         """
         return self._client.close()
 
+    def get_search_index_buffered_sender(self, **kwargs):
+        # type: (dict) -> SearchIndexingBufferedSender
+        """Return a Search Indexing Buffered Sender instance
```

Comment:
There shouldn't be any spaces in that comment, right? It's returning an actual class named `SearchIndexingBufferedSender`.